reco_utils/recommender/vowpal_wabbit/vw.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+This file provides a wrapper to run Vowpal Wabbit from the command line through python.
+It is not recommended to use this approach in production, there are python bindings that can be installed from the
```

Comment:
Why this is not recommended? it should work similarity to the command lines, doesn't it?
I remember you mentioned that the python bindings had some problems...

Reply:
it does, but the extra step of having to write the dataframe to file is really inefficient. the desired approach would be to import the sklearn wrapper that comes with the python bindings. however the current code will not easily install under conda. I'm working on that, but it's not there yet.

once that is working i suggest we move over to using that code and pulling any useful dataframe formatting either into VW itself or into a utility function. Then we can reduce additional wrapper code here.

Reply:
ok I see